mypy.ini

```diff
@@ -73,8 +73,6 @@ disallow_untyped_defs = False
 # forwarders.
 [mypy-lsst.pipe.base.task_logging.*]
```

Comment:
I realize that I deleted this file a while back so can you also please delete this line and the line below?